handshake_server.go

```diff
@@ -879,12 +879,19 @@ func (hs *serverHandshakeState) clientHelloInfo() *ClientHelloInfo {
 		pskBinder = hs.clientHello.psks[0].binder
 	}
 
+	var signatureSchemes []SignatureScheme
+	if hs.clientHello.vers > VersionTLS12 && len(hs.clientHello.supportedSignatureAlgorithmsCert) > 0 {
+		signatureSchemes = hs.clientHello.supportedSignatureAlgorithmsCert
+	} else {
```

Comment:
I don't think this is correct, according to the spec. The `signature_algorithms` and `signature_algorithms_cert` might be disjoint sets.